fmperf/loadgen/collect_energy.py

```diff
@@ -179,21 +180,14 @@ def summarize_energy(start_ts: str):
             all_df[m] = metric_df
 
         # print(all_df)
-        all_df.index.name = "start_time"
         all_df.columns = [
             "num_users",
             "dcgm_idle_power",
             "dcgm_energy",
```

Comment:
we also want to indicate that the energy/power here is the total energy/power, or dynamic energy/power, since we already mentioned `dcgm_idle_power`